video/out/opengl/context.c

```diff
@@ -211,6 +211,15 @@ int ra_gl_ctx_color_depth(struct ra_swapchain *sw)
     return depth_g;
 }
 
+static bool ra_gl_ctx_wait_for_frame(struct ra_swapchain *sw)
+{
+    struct priv *p = sw->priv;
+    bool should_draw = true;
+    if (p->params.wait_for_frame)
+        should_draw = p->params.wait_for_frame(sw->ctx);
+    return should_draw;
```

Comment:
nit: the use of a `bool` here is a bit redundant and verbose, why not just `return p->params.wait_for_frame(sw->ctx)` directly? (and otherwise `return true;`)